osu.Game/Graphics/ScreenshotManager.cs

```diff
@@ -117,11 +118,11 @@ public void OnReleased(KeyBindingReleaseEvent<GlobalAction> e)
 
                 host.GetClipboard()?.SetImage(image);
 
-                string filename = getFilename();
+                (string filename, var stream) = getWritableStream();
 
                 if (filename == null) return;
 
-                using (var stream = storage.CreateFileSafely(filename))
+                using (stream)
```

Comment:
This tuple return isn't gonna work so well whenever NRT is enabled in this file. But I'm not about to block this PR on that.